README.md

```diff
@@ -22,41 +22,90 @@ func sayHelloToCat(kittenName: String) {
 }
+
+sayHello()
+sayHello()
 ```
 
-* This will be the very first time they see a function, be very explicit in your explanation in **detail** to the various parts of the above code. Even go into how *sayHello* is written as camel case. Why are we writing our code within the curly braces `{ }`? If we declare something within those curly braces, can I access that variable outside of the function?
+What's happening here? First you _defined_ the function `sayHello()`, and then you call it twice, on the last two lines of the code above. You should see "Hello there!" printed twice to your playground file's console (don't forget to pop that open by hitting the arrow button at the bottom of the playground!).
 
-* How do I call on a function? When does the code within those curly braces get called? Have them follow along and get used to writing this out in a playground file. 
- 
-* Everyday, our owner takes us for a walk.. in this walk, we have to walk by dozens of cats (we know all of their names) and we want to say Hello to each one and ask them the question.. why do you sleep so much. It would be silly to have to write out doezens of print functions where they only difference is the kitten name. If we can have the student  (or dog) SEE that the only difference between each greeting is the cats name, that is very important. We then introduce **arguments**.
-* At this stage, we go into how to write our own function that includes arguments. Instead of saying "Hello everyone!", we would like to say Hello to each individual cat. Show that in code:
+Does calling a function look familiar? You've been doing that when you write `print("string")` all along!
+
+Notice that once you've defined a function, you can call it as often as you want, and each time the same bit of code will be executed. You could call `sayHello()` five times...fifty times...even a thousand times! Go ahead and try it! (But don't do it a thousand times, because that'll take a long time to type, and there's still quite a bit more of this lesson. You can if you really want to, though—nothing is stopping you!)
+
+## Scopes
+
+Now is a good time to segue very quickly into _scopes_. Every function has its own scope, which means that constants and variables defined in that function are only usable _within_ that function. In `sayHello(), you declared a constant called `greeting`. `greeting` can _only_ be used within the body of `sayHello()`. You can't use it outside of the function.
+
+Don't believe me? Add this line to the end of your playground file:
+
+```swift
+print(greeting)
+```
+
+You should get an error when you try to run your playground!
+
+Scopes are helpful because you can use any name you want for variables and constants inside the function, without having to worry that the name has already been used somewhere else. And other code you write can use the same variable and constant names without worrying about overwriting the variables in your other functions, too. Win-win!
+
```

Comment:
I like explaining this to the student as you did here but I think we expand on this. I would like to start off this paragraph with something like the following which drills home this idea of the visibility of the variable declared in the function.

> Scope refers to the visibility of variables. In other words, which parts of your program can see or use it.  It is very useful to be able to limit a variable's scope to a single function. In other words, the variable will have a limited scope. This way, changes inside the function can't affect other parts of your program in unexpected ways. In our case, we can be certain that since we declared the greeting variable within the scope of our function, no-one outside of that function can mess with it.